base/driver.go

```diff
@@ -0,0 +1,9 @@
+package base
```

Comment:
Could we please merge this one line file into base/kubedriver.go?

Reply:
done